includes/classes/admin/ajax/handlers.php

```diff
@@ -404,6 +405,46 @@ public function gc_dismiss_notice_cb() {
 		wp_send_json_error();
 	}
 
+	/**
+	 * Ajax callback when getting Component subfields for the mapping UI
+	 *
+	 * @since  3.2.20
+	 *
+	 * @return void
+	 */
+	public function gc_component_subfields_cb() {
+		if ( ! $this->_post_val( 'subfields_data' ) ) {
+			wp_send_json_error("one");
```

Comment:
As I'm not fully up to speed with Wordpress what does it mean to send `one` into `json_error`?

Reply:
@CWDN @HughbertD 

We removed the "one" and "two" from this file as that was being used as part of testing to know where the error hit during dev. This is now following the error response pattern found in the rest of the file.